### ISSUE TITLE
Will Tai-e support Android APK analysis?

### ISSUE THREAD

**Issue by fengdai** [pascal-lab/Tai-e#130]
### Clear and concise description of the problem

I have recently made slight modifications to Tai-e to enable it to accept APK files for analyzing an industry-scale Android application, and it has performed remarkably well in certain scenarios. I was wondering if Tai-e has plans to officially support Android APK analysis in the future?

### Intention to submit PR

No

**Comment by ayanamists** [comment]
To analyze Android applications, it's not just about processing the APK files themselves (you can use tools to convert dex to jar, so you can directly input to Tai-e), but more importantly, it's about modeling Android-specific features and dataflows. As far as I know, some people have already started implementing relevant analyses on Tai-e, stay tuned!

**Comment by fengdai** [comment]
> it's about modeling Android-specific features and dataflows

Exactly. Currently, we are using Tai-e to analyze a same portion of our Android application. For instance, we generate CG of a specific method using PTA along with customized PTA plugins (modeling).

> As far as I know, some people have already started implementing relevant analyses on Tai-e, stay tuned!

Could you please provide more information about this work? We also have plan to enhance Tai-e's capability for analyzing Android apps. If it's open source, we would be interested in contributing.

**Comment by 1010353663** [comment]
@fengdai Hi, we are currently supporting Android analysis using Tai-e, focusing on inter-component communication (ICC), callback analysis, entry point analysis, and more. Our work will be published and open-sourced soon.

**Comment by 1010353663** [comment]
@fengdai Hi, we are currently providing comprehensive support for Android analysis using Tai-e, focusing on inter-component communication (ICC), callback analysis, entry point analysis, and more. Our work will be released and open-sourced soon.
